src/components/underground.html

```diff
@@ -19,6 +19,43 @@
        <div class="tab-content">
            <div id="dig" class="tab-pane fade in active show">
                <div class="modal-body" style="width: 80%;margin: auto;">
+
+                   <table class='table table-sm' style="width:60%; margin: auto">
+                       <tbody>
+                           <tr>
+                               <td class="col-md-4">
```

Comment:
Browsers are smart enough to close these `<td>` elements for us, but it would be nice if we did it.